examples/text_input_aligned/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "text_input_aligned"
+version = "0.1.0"
+authors = ["Subroto Biswas <sub14biswas@gmail.com>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+iced = { path = "../..", features = ["async-std", "debug"] }
+log = "0.4.0"
+log4rs = "0.11.0"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+async-std = "1.0"
+
+[package.metadata.deb]
+assets = [
+    ["target/release/todos", "usr/bin/iced-todos", "755"],
+    ["iced-todos.desktop", "usr/share/applications/", "644"],
+]
```

Comment:
This section can be removed. Probably a leftover from copy-pasting :sweat_smile: